README.md

```diff
@@ -15,3 +15,56 @@ undefined
 > parsedScript = nbs.parseRawScript('a914c664139327b98043febeab6434eba89bb196d1af87', 'hex')
 'OP_HASH160 PUSHDATA(20)[c664139327b98043febeab6434eba89bb196d1af] OP_EQUAL'
 ```
+
+The library also exposes several utility functions for getting information about bscript opcodes.
+
+### Conversion between opcodes and asm terms
+```
+> nbs.opcodes.opcodeForWord('OP_EQUAL')
```

Comment:
NIT: We should change `nbs` to `bscript`. I forgot to change it when I changed the package name from `node-bscript-parser`.